pkg/cloudwatch/example/main.go

```diff
@@ -0,0 +1,55 @@
+package main
```

Comment:
IMO, just go ahead and remove `pkg/cloudwatch/example`. Not all that useful.

You can also remove the newly vendored github.com/pborman/uuid package.